Example/Pods/Local Podspecs/Swiftilities.podspec.json

```diff
@@ -38,6 +38,10 @@
       ]
     },
     {
+      "name": "Collections",
```

Comment:
#nth some tests for the `Collections` extension. Probably just pull a random item 1000 times from a smallish collection and make sure it doesn't crash.